scripts/zkai_attention_kv_proof_route_selector_gate.py

```
@@ -422,6 +443,58 @@ def load_risc0_wide_masked_sequence_payload(path: pathlib.Path = RISC0_WIDE_MASK
+        "proof_backend",
+        "proof_backend_version",
+        "semantic_scope",
+        "statement_version",
+    }
+    if set(envelope) != expected_keys:
+        raise AttentionKvRouteSelectorError("Stwo native proof envelope schema drift")
+    input_payload = load_stwo_native_masked_sequence_payload()
+    if envelope.get("input") != input_payload:
+        raise AttentionKvRouteSelectorError("Stwo native proof envelope input drift")
+    if envelope.get("decision") != "GO_STWO_NATIVE_ATTENTION_KV_MASKED_SEQUENCE_AIR_PROOF":
+        raise AttentionKvRouteSelectorError("Stwo native proof envelope decision drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)